request-controller/src/main/java/org/wildfly/extension/requestcontroller/RequestController.java

```diff
@@ -397,20 +397,20 @@ private boolean runQueuedTask(boolean hasPermit) {
     }
 
     private QueuedTask findForcedTask() {
-        QueuedTask task = null;
+        QueuedTask _ret = null;
```

Comment:
I'll let @bstansberry review your changes as he already reviewed your PR

However could you please rename this variable? WildFly code convention do not use `_` to prefix variables? thanks

Reply:
done. 
One additional thing I noticed when I saw this code once again after a while @bstansberry :
There is a potential problem but as I'm not sure what forced task really is I'm not sure is it intended or just another bug: 
This code: 
```java
        while ((task = taskQueue.poll()) != null) {
            if (task.forceRun) {
                forcedTask = task;
            } else {
                storage.add(task);
            }
        }
```
will go trough all tasks in `taskQueue` so there is potential problem of dropping forced tasks if there is more than one. Only last forced task will be returned to a caller, previous ones are not added to a `storage` collection so they are just lost.

Reply:
Apologies @khroolick, I got distracted. This should fix that:

while (forcedTask == null && (task = taskQueue.poll()) != null) {

With that the code looks much as it did before though, with different variable names, the NPE fixed and pretty redundant taskQueue.isEmpty() call removed. If that resolves the issue you were seeing though, it's fine with me.

Reply:
Same on my side, it takes longer than it should :-)
I fully agree with you @bstansberry, applied suggested change and I'm happy to merge it in the current shape.